scribe-org/Scribe-Android
# Refactor Long Lines of Code for Improved Readability

### Contributor checklist




---

### Description

This PR addresses the issue of lines of code exceeding the maximum allowed line length, as specified in our coding standards. Long lines can be hard to read and maintain, so I've refactored the relevant code to improve readability and adhere to the established guidelines.

**Changes Made:**
- Refactored methods and lines of code that exceeded the maximum line length.
- Used string concatenation and multi-line formatting for better clarity and maintainability.

**Example Before:**
```kotlin
fun someLongMethodName(param1: String, param2: Int, param3: Boolean): String {
    val longString = "This is an example of a very long string that exceeds the maximum allowed line length and should be refactored to be more readable and maintainable."
}
```

**Example After:**
```kotlin
fun someLongMethodName(param1: String, param2: Int, param3: Boolean): String {
    val longString = "This is an example of a very long string that " +
                     "is now split into multiple lines to adhere " +
                     "to the maximum allowed line length and improve readability."
}
```

**How to Verify:**
To ensure compliance with the maximum line length rule:
1. Enable the `MaxLineLength` rule in the `detekt.yml` configuration file.
2. Run `./gradlew detekt` in the terminal to identify any remaining lines that violate the rule.

**Additional Information:**
This change significantly enhances the code's readability and maintainability. The documentation provides more insights into the importance of this coding standard.


### Related issue





-   #93
